dev/com.ibm.ws.concurrent/resources/com/ibm/ws/concurrent/resources/CWWKCMessages.nlsprops

```diff
@@ -66,6 +62,10 @@ CWWKC1103.skip.run.failed=CWWKC1103E: Execution of task {0}, which was submitted
 CWWKC1103.skip.run.failed.explanation=The skipRun operation for the Trigger has failed, causing the current execution attempt for the task to be skipped.
 CWWKC1103.skip.run.failed.useraction=Correct the cause of the failure and decide whether or not to resubmit the task.
 
+CWWKC1110.task.canceled=CWWKC1110I: The task {0}, which was submitted to executor service {1}, is canceled.
+CWWKC1110.task.canceled.explanation=Execution of the task has been canceled.
```

Comment:
```suggestion
CWWKC1110.task.canceled.explanation=Execution of the task was canceled.
```

Reply:
Same comment here (moved message, not new), but it should be okay to change an explanation/useraction.